content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -4,185 +4,229 @@ Process receipts and deliveries in two steps
 
 .. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-shipments.png
    :align: center
-   :alt: Set incoming and outgoing shipment options to receive and deliver in two steps.
+   :alt: Incoming and outgoing shipments set to two-step on warehouse form.
 
-.. tip::
-   Activating two-step receipts and deliveries will create new *input* and *output* locations, which
-   by default, are labeled :guilabel:`WH/Input` and :guilabel:`WH/Output`, respectively, on the
-   :guilabel:`Locations` dashboard. To rename these locations, go to :menuselection:`Configuration
-   --> Locations`, and select the :guilabel:`Location` to change. On the location form, update the
-   :guilabel:`Location Name`, and make any other changes (if necessary).
+.. note::
+   Selecting two-step receipts and deliveries automatically creates new *Input* and *Output*
+   warehouse locations in the database, named `WH/Input` and `WH/Output`, respectively.
 
-Process a receipt in two steps (input + stock)
-==============================================
+   To rename or edit these locations, navigate to :menuselection:`Inventory app --> Configuration
+   --> Locations`, and select the desired location.
 
-Create a purchase order
------------------------
+   On the location's form, change the :guilabel:`Location Name`, and make any other necessary
+   changes.
 
-On the main :menuselection:`Purchase` application dashboard, start by making a new quote by clicking
-:guilabel:`New`. Then, select (or create) a :guilabel:`Vendor` from the drop-down field, add a
-storable :guilabel:`Product` to the order lines, and click :guilabel:`Confirm Order` to finalize the
-quote as a new purchase order.
+Process receipt in two steps (input + stock)
+============================================
 
-.. important::
-   For businesses with multiple warehouses that have different step configurations, the
-   :guilabel:`Deliver To` field on the :abbr:`PO (Purchase Order)` form may need to be specified as
-   the correct *input location* connected to the two-step warehouse, which can be done by selecting
-   the warehouse from the drop-down selection that includes the `Receipts` label at the end of the
-   name.
+When products are received in two steps, they move from the vendor location to an input location,
+then from the input location to warehouse stock in the database upon validation of a purchase order
+(PO) and a subsequent internal transfer.
+
+Create purchase order
+---------------------
 
-After confirming the :abbr:`PO (Purchase Order)`, a :guilabel:`Receipt` smart button appears at the
-top of the :abbr:`PO (Purchase Order)` form. Click it to reveal the new receipt (WH/IN) operation.
+To create a |PO|, navigate to the :menuselection:`Purchase app`, and click :guilabel:`New`. This
+opens a blank :guilabel:`Request for Quotation` (RfQ) form.
+
+Add a vendor in the :guilabel:`Supplier` field. Then, fill out the various fields on the |RfQ|, as
```

Comment:
In 17.2 the field is changed from Vendor to "Supplier" in this instance

Reply:
I, too, am only seeing a 'Vendor' field. It also seems to appear that way in 17.2 -- at least in the 17.2 runbot I used to double-check this. It might be worth looking into the accuracy of this field name during your revisions.

![Screenshot 2024-05-06 100913](https://github.com/odoo/documentation/assets/73958186/e90c16bf-7db4-4ad8-9a5a-425e10a6458b)

Reply:
I now am also seeing this. Not sure what happened to a few of the field names I saw upon first writing this... making the according changes now